src/main/java/me/jellysquid/mods/sodium/client/render/chunk/RegionChunkRenderer.java

```diff
@@ -127,38 +128,47 @@ private boolean buildDrawBatches(List<RenderSection> sections, BlockRenderPass p
                 continue;
             }
 
-            ChunkRenderBounds bounds = render.getBounds();
-
             long indexOffset = state.getIndexSegment()
                     .getOffset();
 
             int baseVertex = (int) state.getVertexSegment()
                     .getOffset();
 
-            this.addDrawCall(state.getModelPart(ModelQuadFacing.UNASSIGNED), indexOffset, baseVertex);
+            if (SodiumClientMod.options().advanced.useBlockFaceCulling) {
```

Comment:
This should be moved to a `private final` variable, and initialized once when `RegionChunkRenderer` is created. If this setting changes, the renderer will be re-created, so it should be safe.

Reply:
I've done this change in my latest commit. thank you